### DISCUSSION TITLE
NoahMP TWS output

### DISCUSSION THREAD

**Discussion by cbachand** [NASA-LIS/LISF#1499] [Q&A] [answered]
Hi, is Noah-MP able to output TWS? I requested TWS output in my model_output.txt file, but it was not produced. Is there a specific setting I need to turn on in Noah-MP for this to work?

**Comment by dmocko** [comment, marked as answer]
Hi @cbachand

Yes, both Noah-MP-4.0.1 and Noah-MP-3.6 are able to output TWS.

Do you have a TWS field in your output that is all zeroes, or does TWS not appear as an output field?

The TWS line for your MODEL OUTPUT TBL file should be:

`TWS:          1  mm      -    1 0 0 1 175 1       # Terrestrial Water Storage (mm)`

Only units of "mm" are available as an output for TWS.

If this still doesn't work, please attach your lis.config file, your lislog.0000 file, and your MODEL OUTPUT TBL file.
